app/views/partners/requests/index.html.erb

```diff
@@ -18,7 +18,7 @@
           <li class="breadcrumb-item"><a href="#">Essentials Requests</a></li>
         </ol>
       </div>
-    </div>`
+    </div>
```

Comment:
Good catch. Always important to check for backticks after tromping around through code.